### PR TITLE
 ZhukovVA lab1

### PR DESCRIPTION
**Pull request by Heiwy**

### CODE REVIEW

File: ZhukovVA/lab_1/main.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	int esc;
+
+	do {
+		system("cls");
+		cout << "Type two numbers and an operator in the following format 'Number_1 <Operator> Number_2': ";
+		cin >> num1 >> op >> num2;
+
+		switch (op) {
+		case '+':
+			cout << "Result: " << num1 + num2;;
```
Comment (Toxa-man) [inline ZhukovVA/lab_1/main.cpp]: Please extract `cout << "Result: " << num1 ? num2;` lines from switch..case in order to reduce code duplication

**Review by Toxa-man — COMMENTED** [review]
Code issues

**Comment by Toxa-man** [comment]
Please give meaningful names to variables

**Comment by Heiwy** [comment]
I have made the corrections.

**Pull request merged**